ci/publish-tfgen-package

```diff
@@ -97,4 +97,10 @@ if [[ "${TRAVIS_OS_NAME:-}" == "linux" ]]; then
 	    --skip-existing \
             --verbose
     fi
+
+    # Finally, publish the NuGet package if any exists.
+    if [ -n "${NUGET_PUBLISH_KEY}" ]; then
+        find "${SOURCE_ROOT}/sdk/dotnet/bin/Debug/" -name '*.nupkg' \
```

Comment:
I should point out that this isn't super safe if it's ever done on a machine that may have stale builds in the `bin/Debug` folder.

Reply:
from yoru feedback in pulumi/pulumi perhaps we should also filter this down to `Pulumi*.nupkg`?  just to make sure we don't somehow have and publish nupkgs from others there?

Reply:
That'll make things a bit safer. I'll make the change.